src/h3lib/lib/algos.c

```diff
@@ -37,9 +37,12 @@
 #include "polygon.h"
 #include "vertexGraph.h"
 
-/*
- * Return codes from gridDiskUnsafe and related functions.
+/**
+ * TODO: Move to h3api.h.in
+ * TODO: Use correct codes from RFC
  */
+#define FLAG_ANY_VERTEX 1
+#define FLAG_ALL_VERTEXES 2
```

Comment:
Note that the RFC doesn't actually include these flags, which are less rigorous than "contains" or "intersects". So that gives us minimum 5 modes at the outset.

Also, should this be an enum?